odoo/tools/pdf.py

```diff
@@ -177,7 +189,7 @@ def getAttachments(self):
             return []
 
 
-class OdooPdfFileWriter(PdfFileWriter):
+class OdooPdfWriter(PdfWriter):
```

Comment:
Renaming is maybe a good idea for the future, but if anyone is using it in stable we may break that/

I would at least add a `OdooPdfWriter = PdfWriter`  and maybe add a deprecation warning in master

Reply:
You could also make `OdooPdfFileWriter(PdfWriter)`. This way the import of Odoo would not break